doc/feature/restAPI.md

```diff
@@ -0,0 +1,129 @@
+Publishing the REST API exposed by the HTTP Server
+============
+
+The http server exposes C++ objects via a REST API, in a JSON serialized format.
+The purpose of this document is to describe how this API request is implemented,
+and invoked.
```

Comment:
The second sentence is not clear to me. At a first glance I understand something different from what I think it means. What this document describes is how to request the specification of services provided by the server, isn't it?